### PR TITLE
Fix a few minor spelling mistakes

### PR DESCRIPTION
**Pull request by parlough**

### CODE REVIEW

File: dwds/debug_extension/web/background.dart [hunk 1]
```diff
@@ -570,7 +570,7 @@ Future<void> _startSseClient(
         'Authentication required.\n\nClick OK to authenticate then try again.',
       )) {
         // TODO(grouma) - see if we can get a callback on a successful auth
-        // and automatically reinitiate the dev workflow.
+        // and automatically initiate the dev workflow.
```
Comment (elliette) [inline dwds/debug_extension/web/background.dart:573]: I'm guessing reinitiate was the intended word here

Comment (parlough) [reply dwds/debug_extension/web/background.dart]: After looking further into the context, that makes sense. Reverted!

Thanks for looking so closely :)

**Review by annagrin — APPROVED** [review]
LGTM!

**Comment by elliette** [comment]
Thanks! I'll working on merging this and the other open PRs once the DWDS/WebDev releases have been completed

**Pull request merged**